### ISSUE TITLE
does openh264 encoder parameter iComplexityMode work

### ISSUE THREAD

**Issue by hjybdrs** [cisco/openh264#3129]
version:1.8
change different iComplexityMode to verify video output,  but the bps and frameRate didn't have remarkable change. whether 1.8 have any implementions on iComplexityMode?

**Comment by huili2** [comment]
You are right. iComplexityMode does not have too much difference now. It will be used in the future.

**Comment by hjybdrs** [comment]
> You are right. iComplexityMode does not have too much difference now. It will be used in the future.
whether different IcomplexityMode influence CPU usageRate?

**Comment by hjybdrs** [comment]
> You are right. iComplexityMode does not have too much difference now. It will be used in the future.
i found there are several code in code like blow , confusing whether the different funcion invoke influence the pic output or CPURate?

`static inline void SetFastCodingFunc (SWelsFuncPtrList* pFuncList)()
static inline void SetNormalCodingFunc (SWelsFuncPtrList* pFuncList)()
if (bFastMode) {
      pFuncList->pfCalculateSatd = NotCalculateSatdCost;
      pFuncList->pfInterFineMd = WelsMdInterFinePartitionVaa;
    } else {
      pFuncList->pfCalculateSatd = CalculateSatdCost;
      pFuncList->pfInterFineMd = WelsMdInterFinePartition;
    }`

**Comment by huili2** [comment]
Yes, slightly different, not exactly the same.
